src/components/release-cards.tsx

```diff
@@ -8,31 +8,36 @@ interface Props {
 export default function ReleaseCards({ line }: Props): JSX.Element {
   const fileName = line && line.version;
   return (
-    <div>
-      <div>
+    <div className="release-card-container">
```

Comment:
Lets use some more semantic elements here :) 
https://bitsofco.de/sectioning-content-in-html5/

I'd say `section` for the card containers and `<h1>` / `<h2>` for the lines of text.

Reply:
Also, release-card seems like it can be pulled out to its own functional component!

Reply:
Agreed